modules/pricer/src/main/java/com/opengamma/strata/pricer/bond/DiscountingFixedCouponBondTradePricer.java

```diff
@@ -424,8 +435,14 @@ public Payment upfrontPayment(ResolvedFixedCouponBondTrade trade) {
   }
 
   //-------------------------------------------------------------------------
-  // calculate the settlement date
-  private LocalDate settlementDate(ResolvedFixedCouponBondTrade trade, LocalDate valuationDate) {
+  /**
+   * Calculates the settlement date.
```

Comment:
Add that in absence of settlement date (position?), the valuation date is returned.

Reply:
Javadoc updated